File: src/main/java/com/clearcont/clearcontapp/helpers/DecimalFormatBR.java
package com.clearcont.clearcontapp.helpers;

import org.jetbrains.annotations.NotNull;

import java.text.DecimalFormat;
import java.text.DecimalFormatSymbols;

public class DecimalFormatBR {
    public static @NotNull DecimalFormat getDecimalFormat() {
        DecimalFormat formatter = new DecimalFormat("R$ #,###.##");
        DecimalFormatSymbols s = new DecimalFormatSymbols();
        s.setDecimalSeparator(',');
        s.setGroupingSeparator('.');
        formatter.setDecimalFormatSymbols(s);
        return formatter;
    }
}

File: src/main/kotlin/br/com/clearcont/clearcontwebapp/helpers/DecimalFormatBR.kt
package br.com.clearcont.clearcontwebapp.helpers

import java.text.DecimalFormat
import java.text.DecimalFormatSymbols

object DecimalFormatBR {
    val decimalFormat: DecimalFormat
        get() {
            val formatter = DecimalFormat("R$ #,###.##")
            val s = DecimalFormatSymbols()
            s.decimalSeparator = ','
            s.groupingSeparator = '.'
            formatter.decimalFormatSymbols = s
            return formatter
        }
}
